dev-util/phpstorm/phpstorm-9.0.2.141.2462.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit eutils versionator
+
+SLOT="$(get_major_version)"
+MY_PV="$(get_version_component_range 4-5)"
```

Comment:
This is also used in one place only. Better to inline it since MY_PV suggests it's about the filename actually.

Or even better, you could drop the two extra version components from filename, and inherit `vcs-snapshot` to handle the weird in-tarball directories for you. Then you don't need to adjust `S` at all.